ANMN/LTSP/TSaggregator/aggregated_timeseries.py

```diff
@@ -179,29 +179,66 @@ def set_variableattr(varlist, variable_attribute_dictionary, add_variable_attrib
     return variable_attributes
 
 
-def generate_netcdf_output_filename(fileURL, nc, VoI, file_product_type, file_version):
+def get_data_code(VoI):
+    """
+    get data code sensu IMOS conventions from variable code
+
+    :param VoI: variable code
+    :return: variable data code
+    """
+
+    #dictionary of data code. could be read from external file
+    dataCodes = {'DEPTH':       'Z',
+                 'PRES':        'Z',
+                 'PRES_REL':    'Z',
+                 'TEMP':        'T',
+                 'PSAL':        'S',
+                 'PAR':         'F',
+                 'TURB':        'U',
+                 'DOX1':        'O',
+                 'DOX1_2':      'O',
+                 'DOX1_3':      'O',
+                 'DOX2':        'O',
+                 'DOX2_1':      'O',
+                 'DOXS':        'O',
+                 'CPHL':        'B',
+                 'CHLU':        'B',
+                 'CHLF':        'B'}
+    return dataCodes[VoI]
+
+def get_facility_code(fileURL):
+    """
+    get the facility code from the file URL
+
+    :param fileURL: URL of a file
+    :return: facility code
+    """
+    split_filename = fileURL.split("/")[-1].split("_")
```

Comment:
`os.path.basename(fileURL)` would be a bit more robust than `fileURL.split("/")[-1]`

Reply:
ok.